### PR TITLE
Improve UI appearance

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: application/dash_application/views/pre_exome.py [hunk 1]
```diff
@@ -191,7 +191,9 @@ def generateDebugLine(click, runs, firstsort, secondsort, colourby,
 layout = html.Div(className='body',
     children=[
         navbar("Pre-Exome"),
-        html.Div(className='sidebar',
+        html.Div(className='row flex-container',
```
Comment (avarsava) [inline application/dash_application/views/pre_exome.py]: Could there be an explanation (or link to explanation) of these class names somewhere?

**Comment by avarsava** [comment]
This is going to sound insane, but in rshiny that 'terminal output' textarea expands to the height of its contents. Could we do that here?

**Comment by avarsava** [comment]
Also could the sidebar be as long as it needs to be to show all the controls rather than scrolling?

File: application/dash_application/views/pre_exome.py (hunk 1 above)
Comment (apmasell) [reply application/dash_application/views/pre_exome.py]: Honestly, I copied it from the template. I'm not entirely sure what these really do.

**Comment by apmasell** [comment]
This should address everything except the loading problem, which seems to be a client-side issue where React is clearing the loading message and then spending forever rendering stuff.

**Pull request merged**